docker/airflow/entrypoint.sh

```diff
@@ -34,6 +34,21 @@ function header() {
 if [ "$1" == help ] || [ "$1" == --help ]; then help_text && exit 0; fi
 sleep 0.1;  # The $COLUMNS variable takes a moment to populate
 
+# Reformat Airflow connections that use https
+header "MODIFYING ENVIRONMENT"
+# Loop through environment variables, relying on naming conventions
+while IFS="="; read var_name var_value; do
+    echo "Variable Name : $var_name"
+    echo "Original Value: $var_value"
+    # call python to get the new value
+    url_encoded=`python -c"from urllib.parse import quote_plus; import sys; print(quote_plus(sys.argv[1]))" $var_value`
+    new_value='https://'$url_encoded
+    echo "New Value:      $new_value"
+    # set the environment variable
+    export $var_name=$new_value
+done < <(env | grep "AIRFLOW_CONN.*https.*")
```

Comment:
I think we may need to include regular `http` links (I use that locally for the docker driver). Also we'll probably want to make sure the `http` value is right after the `=`, and account for potential `"` characters wrapping the values.
```suggestion
done < <(env | grep -E "^AIRFLOW_CONN[A-Za-z_]+=\"?http.*")
```

Reply:
It seems like (airflow really recommends all caps for the variable names)[https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html#storing-a-connection-in-environment-variables], so I would think that we wouldn't need to `a-z` here, but then I saw (AIRFLOW_CONN_AWS_DEFAULT)[https://github.com/WordPress/openverse-catalog/blob/b44f3398d89450c6bfe765e03c832c241d5067c5/env.template#L44] and got that you must mean to include http things within the longer connection string. Am I understanding that correctly? 

If so, I think I need to drop the IFS shortcut and use regex so that we don't url encode more than we want to, right? Like this is _not_ what we want, right?
```
>>> from urllib.parse import quote_plus
>>> print(quote_plus('my-conn-type://login:password@host:port/schema?param1=val1&param2=val2'))
my-conn-type%3A%2F%2Flogin%3Apassword%40host%3Aport%2Fschema%3Fparam1%3Dval1%26param2%3Dval2 
>>> print(quote_plus('aws://test_key:test_secret@?region_name=us-east-1&host=http://s3:5000'))
aws%3A%2F%2Ftest_key%3Atest_secret%40%3Fregion_name%3Dus-east-1%26host%3Dhttp%3A%2F%2Fs3%3A5000
```

Reply:
Yes, that's exactly right. The second case is what I was trying to avoid with this `grep`, because it _has_ `http` in it but is not the connection type that we want to change. Great point about casing, I think we could totally exclude `a-z` here.

Reply:
Ah! Just seeing this now. OK, more in a second.